app/views/tas/_upload_modal.html.erb

```diff
@@ -0,0 +1,33 @@
+<%= content_for :modal_id, 'upload_dialog' %>
+<%= content_for :modal_title, I18n.t('upload') %>
+<%= content_for :modal_content do %>
+
+    <!-- Script for disabling upload buttons. -->
+    <%= javascript_include_tag 'upload_button_control.js',
+                               id: 'upload_script',
+                               upload_id: 'userlist',
+                               button_id: 'upload' %>
+
+    <p><%= I18n.t('graders.upload.select_csv_file').html_safe %></p>
+    <p><%= I18n.t('graders.upload.grader_same_user') %></p>
+
+    <%= form_tag upload_ta_list_tas_path, { multipart: true } %>
+    <div>
+      <label for='userlist'><%= I18n.t('graders.upload.csv_file') %></label>
```

Comment:
Use `label_tag` and `file_field_tag` (http://api.rubyonrails.org/classes/ActionView/Helpers/FormTagHelper.html#method-i-file_field_tag).